### ISSUE TITLE
Bug Fix Logger

### ISSUE THREAD

**Issue by Deepakraj8055** [amosproj/amos2022ws01-firmware-scraper#96]
## User story
1. As a developer
2. I want the logger to log only once per log
3. So that the bug with doubled logging is gone

## Acceptance criteria
* There's only one line for logging, instead of doubled
* Logger runs without any bugs

## Definition of done (DoD)
* All the acceptance criteria have been met
* The code has been merged with the main one
* The documentation and Bill of Materials have been updated
* The code is complete and uploaded in GitHub
* Functionality has been reviewed

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
